lib/src/components/NxTree/NxTreeItem.tsx

```diff
@@ -136,12 +131,12 @@ export default function NxTreeItem(props: ItemProps) {
   function onKeyDown(evt: KeyboardEvent<HTMLLIElement>) {
     switch (evt.key) {
       case 'ArrowUp':
-        stopPropAndPreventDefault(evt);
```

Comment:
Won't this result in the page scrolling when the user presses the Up arrow here? (and similarly for the other directions)